### PR TITLE
Update SyncCompleteProcessor.cs

### PR DESCRIPTION
**Pull request by PhantomGamers**
I had accidentally done it so we  were sending the planet id as the star id argument, whoops.

### CODE REVIEW

File: NebulaNetwork/PacketProcessors/Session/SyncCompleteProcessor.cs [hunk 1]
```diff
@@ -16,7 +16,7 @@ public class SyncCompleteProcessor : PacketProcessor<SyncComplete>
 
         public SyncCompleteProcessor()
         {
-            playerManager = MultiplayerHostSession.Instance?.PlayerManager;
```
Comment (hubastard) [inline NebulaNetwork/PacketProcessors/Session/SyncCompleteProcessor.cs:19]: Isn't this doing the same as it was previously?

Comment (PhantomGamers) [reply NebulaNetwork/PacketProcessors/Session/SyncCompleteProcessor.cs:19]: VS warns against using null propagation on unity objects as they can be invalid without being null

Comment (PhantomGamers) [reply NebulaNetwork/PacketProcessors/Session/SyncCompleteProcessor.cs:19]: https://blog.unity.com/technology/custom-operator-should-we-keep-it

**Review by hubastard — APPROVED** [review]

File: NebulaNetwork/PacketProcessors/Session/SyncCompleteProcessor.cs (hunk 1 above)
Comment (Therzok) [reply NebulaNetwork/PacketProcessors/Session/SyncCompleteProcessor.cs:19]: Yeah, `?.` on anything defined in unity is dangerous.

**Pull request merged**